node/src/container_chain_spawner.rs

```diff
@@ -550,7 +552,6 @@ fn handle_update_assignment_state_change(
         if let Some(para_id) = current {
             // Check if we get assigned to orchestrator chain
             if para_id == orchestrator_para_id {
-                call_collate_on = Some(collate_on_tanssi);
```

Comment:
So, is this just empty if?

Reply:
Good catch haha

Reply:
Also I believe the logic of `need_to_restart_current` and `need_to_restart_next` can be simplified, but I didn't want to touch it in case it breaks something.

Reply:
let's leave it as a future improvement yes